src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Services/ImporterService.cs

```diff
@@ -185,7 +185,7 @@ public SubjectMeta GetMeta(DataTable table, Subject subject, StatisticsDbContext
             await InsertObservations(context, observations);
         }
 
-        public static GeographicLevel GetGeographicLevel(IReadOnlyList<string> line, List<string> headers)
+        public GeographicLevel GetGeographicLevel(IReadOnlyList<string> line, List<string> headers)
```

Comment:
Just wondering if  these methods turning from static into instance methods is a hangover of adding logging and then removing it, or something like that?

Reply:
It was exactly that!  I figured I didn't see the harm in leaving them as instance methods